doc/api/dns.markdown

```diff
@@ -257,47 +294,36 @@ Each DNS query can return one of the following error codes:
-found, then return IPv4 mapped IPv6 addresses. Note that it is not supported
-on some operating systems (e.g FreeBSD 10.1).
-
 ## Implementation considerations
 
-Although [`dns.lookup()`][] and `dns.resolve*()/dns.reverse()` functions have
-the same goal of associating a network name with a network address (or vice
-versa), their behavior is quite different. These differences can have subtle but
-significant consequences on the behavior of Node.js programs.
+Although [`dns.lookup()`][] and the various `dns.resolve*()/dns.reverse()`
+functions have the same goal of associating a network name with a network
+address (or vice versa), their behavior is quite different. These differences
+can have subtle but significant consequences on the behavior of Node.js
+programs.
 
-### dns.lookup
+### `dns.lookup()`
 
 Under the hood, [`dns.lookup()`][] uses the same operating system facilities
 as most other programs. For instance, [`dns.lookup()`][] will almost always
 resolve a given name the same way as the `ping` command. On most POSIX-like
 operating systems, the behavior of the [`dns.lookup()`][] function can be
-tweaked by changing settings in `nsswitch.conf(5)` and/or `resolv.conf(5)`, but
-be careful that changing these files will change the behavior of all other
-programs running on the same operating system.
-
-Though the call will be asynchronous from JavaScript's perspective, it is
-implemented as a synchronous call to `getaddrinfo(3)` that runs on libuv's
-threadpool. Because libuv's threadpool has a fixed size, it means that if for
-whatever reason the call to `getaddrinfo(3)` takes a long time, other
-operations that could run on libuv's threadpool (such as filesystem
+modified by changing settings in `nsswitch.conf(5)` and/or `resolv.conf(5)`,
+but note that changing these files will change the behavior of _all other
+programs running on the same operating system_.
+
+Though the call to `dns.lookup()` will be asynchronous from JavaScript's
+perspective, it is implemented as a synchronous call to `getaddrinfo(3)` that
+runs on libuv's threadpool. Because libuv's threadpool has a fixed size, it
+means that if for whatever reason the call to `getaddrinfo(3)` takes a long
+time, other operations that could run on libuv's threadpool (such as filesystem
 operations) will experience degraded performance. In order to mitigate this
 issue, one potential solution is to increase the size of libuv's threadpool by
-setting the 'UV_THREADPOOL_SIZE' environment variable to a value greater than
-4 (its current default value). For more information on libuv's threadpool, see
+setting the `'UV_THREADPOOL_SIZE'` environment variable to a value greater than
+`4` (its current default value). For more information on libuv's threadpool, see
 [the official libuv documentation][].
 
-### dns.resolve, functions starting with dns.resolve and dns.reverse
+### `dns.resolve()`, `dns.resolve*()` and `dns.reverse()`
```

Comment:
The `*` is there to act as a wild card... referring to all of the various `dns.resolve*` type methods.